spec/features/availability_spec.rb

```diff
@@ -329,5 +329,9 @@
         expect(page).not_to have_link 'Copy unavailable, request via Interlibrary Loan'
       end
     end
+
+    describe 'News & Microforms Scan Request' do
+      it 'display the correct ILL url'
```

Comment:
Why a pending spec here again? Because you plan to write the sinatra fake?

Reply:
yes, I didn't want to add new fixtures to our sample extract just for these cases. I would need to ask for a new extract for our sample with catkeys for these cases, and then they might change even 🤔

Reply:
ok, marking this as part of #631